core/src/services/s3/backend.rs

```diff
@@ -56,7 +56,7 @@ static ENDPOINT_TEMPLATES: Lazy<HashMap<&'static str, &'static str>> = Lazy::new
     m
 });
 
-const DEFAULT_WRITE_MIN_SIZE: usize = 8 * 1024 * 1024;
+const DEFAULT_WRITE_MIN_SIZE: usize = 5 * 1024 * 1024;
```

Comment:
Hi, 8MiB is the part size recommanded by AWS.

Reply:
I got that

Reply:
let's close it